extensions/vscode/src/extension.ts

```diff
@@ -176,6 +176,7 @@ export const extensionSettings = {
       "defaultConnectServer",
     );
 
+    // For RStudio and Posit Workbench users, look here as a final step
```

Comment:
Feel free to disregard if I'm missing something, but I don't think this is relevant for RStudio, right?

```suggestion
    // For Posit Workbench users, look here as a final step
```

Reply:
The file `rsession.conf`'s `default-rsconnect-server` setting is also in open-source, so it will potentially help RStudio users. I realized that when I started documenting it.